pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -110,12 +122,18 @@ type JaegerSpec struct {
 // +k8s:openapi-gen=true
 type JaegerStatus struct {
 	Version string `json:"version"`
+
+	// +kubebuilder:validation:Default=Pending
```

Comment:
If the default is `Pending`, do we need a const to represent that value? May not be directly used, but for completeness?

Reply:
I'll remove this, it's kinda of misleading. This field is not being used in the current version of the operator-sdk and its generators. I thought about adding it here for the future, but it will probably cause confusion until it's effective.